static/js/services/http-wrapper.js

```diff
@@ -32,13 +37,13 @@ var _ = require('underscore');
     '$http', '$q', 'ActiveRequests',
     function($http, $q, ActiveRequests) {
 
-      var wrap = function(args, fn) {
+      var wrap = function(args, fn, updateTarget) {
```

Comment:
Well think of it as a different name than `left` and `right`. Basically the `left` for the Messages tab is the `messages` route, and the `right` is `messages.detail`. So in my mind using the route is a more descriptive naming system. In addition it's one we can query using the new state. Furthermore it's more generic so will still work if we decide to have a further level of nesting, say `messages.detail.info`.